### PR TITLE
Making some Configuration Edits.

### PR DESCRIPTION
**Pull request by sindhuragrandhi**
Signed-off-by: Sindhura Grandhi <sindhura.grandhi@intel.com>

### CODE REVIEW

File: source/developer-guides/configuration.rst [hunk 1]
```diff
@@ -203,6 +203,10 @@ folder.
 Run this Python\* script in the command window: **CfgDataStitch.py -h**
 for parameters for this script.
 
+Example:
+python CfgDataStitch.py -i sbl_ifwi_old.bin -k ConfigTestKey_Priv_RSA3072.pem -s SblOpen\BootloaderCorePkg\Tools -c .
```
Comment (rprangar) [inline source/developer-guides/configuration.rst:207]: When providing command examples, can you use the format specifier (::)? Please refer to other examples in the same file.

Comment (sindhuragrandhi) [reply source/developer-guides/configuration.rst]: Got it

**Review by rprangar — APPROVED** [review]
Looks good.

File: source/specs/config.rst [hunk 2]
```diff
@@ -953,7 +953,7 @@ Example::
 LENGTH
 ^^^^^^
 
-Length is used to specify the size of the configuration option. Length can also specify size in
+Length is used to specify the size of the configuration option in bytes. Length can also specify size in
```
Comment (rprangar) [inline source/specs/config.rst:956]: Length can also be specified

Comment (sindhuragrandhi) [reply source/specs/config.rst]: sure

**Review by rprangar — APPROVED** [review]

**Review by rprangar — APPROVED** [review]

**Pull request merged**